packages/experiments-realm/blog-post.gts

```diff
@@ -607,6 +612,44 @@ export class BlogPost extends CardDef {
     return this.lastUpdated ? this.lastUpdated.toISOString() : undefined;
   }
 
+  get formattedAuthors() {
+    const authors = this.authors ?? [];
+    if (authors.length === 0) return undefined;
+
+    const titles = authors.map((author) => author.title);
+
+    if (titles.length === 2) {
+      return `${titles[0]} and ${titles[1]}`;
+    }
+
+    return titles.length > 2
+      ? `${titles.slice(0, -1).join(', ')}, and ${titles.at(-1)}`
+      : titles[0];
+  }
+
+  get displayAuthors() {
+    return this.authors && this.authors.length > 0;
+  }
+
+  additionalTextForAuthorComponent = (authorIndex: number) => {
+    let isLast = this.authors?.length - 1 === authorIndex;
+    let isLastTwo = this.authors?.length - 2 === authorIndex;
+    let text = '';
+    if (this.authors.length > 2 && !isLast) {
+      text = text + ',';
+    }
+
+    if (this.authors.length > 2) {
+      text = text + ' ';
+    }
+
+    if (isLastTwo) {
+      text = text + 'and';
+    }
+
+    return text;
+  };
+
```

Comment:
Was this in the design, I mean commas and "and"? Adding "," and "and" is not a good way of delimiting items where there are images or website elements involved. I think it's unnecessary and it should only be used in prose or copy.

<img width="801" alt="image" src="https://github.com/user-attachments/assets/f3afcd55-84db-41f7-9895-6dbbcb0e9a6e" />

Reply:
We don't have design for this but the commas and "and" added based on discussion with @lukemelia in our 1-on-1 session.

Reply:
I see, I think it would look better without the noise.

<img width="746" alt="image" src="https://github.com/user-attachments/assets/445a9560-a7c8-4a78-ab66-fc00b651b976" />

Reply:
OK, let's go with @jurgenwerk's suggestion